app/constants/sampleData/index.js

```diff
@@ -2,10 +2,15 @@ import singleSeries from './singleSeries.csv';
 import countriesByYear from './countriesByYear.csv';
 import emissionsPerCountry from './emissionsPerCountry.csv';
 import stockMarkets from './stockMarkets.csv';
+import multibarHorizontalTest from './multibarHorizontalTest.csv';
```

Comment:
How does this work? Importing a csv file? How does it interpret the data in `multibarHorizontalTest`?

Reply:
```